### DISCUSSION TITLE
bitcoin core blocks synced but unable to index ord

### DISCUSSION THREAD

**Discussion by VlVEK** [ordinals/ord#1664] [Q&A]
![image](https://user-images.githubusercontent.com/9716633/218238531-efe3f96f-292a-4eb8-8976-60224b406642.png)

getting this error of unable to index at block 183024 though core is synced fully until current block and running with sparrow

would this need a reinstall of core?

**Comment by MaxflowO2** [comment]
using this in .bitcoin/bitcoin.conf?
```
rpcuser=
rpcpassword=
```
been running this just fine minus 109/107 issues
```
txindex=1
```

**Comment by VlVEK** [comment]
using these in bitcoin.conf file

server=1
txindex=1
rest=1
rpcuser=
rpcpassword=

doesn't solve the issue 

ord index still stuck at block 183024

**Reply by say-sar** [reply]
try going to the folder where the bitcoind app is and run cmd from it. I was able to fix my index issue when i ran this in the command:
bitcoind -txindex=1 -prune=0
